Pages/Language_Page.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using OnboardingSpecflowProject.Utilities;
+using OpenQA.Selenium.DevTools.V124.Debugger;
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using System.Reflection.Emit;
+using System.Security.Policy;
+using TechTalk.SpecFlow;
+
+namespace OnboardingSpecflowProject.Pages
+{
+    internal class Language_Page : CommonDriver 
+    {
+        private string e_message = "//div[@class='ns-box-inner']";
+        //private static IWebElement Language_name = driver.FindElement(By.Name("name"));
+        //public static IWebElement deleteButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]/i"));
+        //public static IWebElement Level_option = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[2]/select/option[2]"));
+        //private static IWebElement Level_option = driver.FindElement(By.XPath($"//div[@class='five wide field']/select[@name='level']/option[@value='Basic']"));
+        //private IWebElement language = driver.FindElement(By.XPath("e_language"));
+        //private string e_language = "//div[@data-tab='first']//tbody[(last)/tr/td[3]/span[1]/i";
+        public void AddLanguage_Level(String language,String level)
```

Comment:
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html